test/Faker/Provider/PhoneNumberTest.php

```diff
@@ -11,10 +11,13 @@
  */
 final class PhoneNumberTest extends TestCase
 {
-    public function testPhoneNumberFormat()
+    public function testE164PhoneNumberFormat()
     {
-        $number = $this->faker->e164PhoneNumber();
-        self::assertMatchesRegularExpression('/^\+[0-9]{11,}$/', $number);
+        for ($i = 0; $i < 1000; ++$i) {
+            $number = $this->faker->e164PhoneNumber();
+            self::assertMatchesRegularExpression('/^\+[1-9]\d{1,14}$/', $number);
+            self::assertLessThanOrEqual(16, strlen($number)); // +\d{2,15}
```

Comment:
What's the benefit of having this comment here?

Reply:
Explaining why `16` as the original E164 is maxed 15 but we have the plus-sign before.
I can remove it but I know how hard it is to understand these numbers sometime later

Reply:
How about writing something like '+ sign and max. 15 digits'? Not having to parse a regex decreases the cognitive load required somewhat IMO.

Reply:
Good point! 👍
https://github.com/FakerPHP/Faker/pull/264/commits/f93d97bb9793171f0212eb2d70725fe0a70cf66f